site: Apache Iceberg Project News and Blog (#15013)
apache/iceberg: site/docs/blog/posts/2026-01-10-iceberg-summit.md

- **Compiling best practices** from real-world deployments
- **Working on ecosystem integrations** with other technologies
- **Sharing thought leadership** about the future of data architecture

**We want to hear from YOU!**

### Session Formats

We're accepting several types of submissions:

- **Breakout Sessions** (30 minutes) - The standard conference talk format
- **Lightning Talks** (15 minutes) - Quick, digestible insights
- **Keynotes** (30 minutes) - Forward-thinking and thought leadership
- **Panels** (45 minutes) - Multi-perspective discussions with 4-6 panelists
- **Workshops/Labs** (60-180 minutes) - Hands-on, interactive sessions

### Topics We're Looking For

- **Iceberg in Production**: Real-world case studies, implementation journeys, lessons learned
- **Iceberg Technology and Features**: Deep dives into Iceberg, PyIceberg, Puffin, and upcoming enhancements
- **Data Architecture**: Stack options, integration patterns, and decision criteria
- **Best Practices**: Tips, tricks, and techniques for specific use cases
- **Thought Leadership**: Your vision for data architecture in the age of big data, cloud, and AI

### Submit Your Talk

> Head over to [Sessionize](https://sessionize.com/iceberg-summit-2026/) to submit your proposal. The Selection Committee will review all submissions to curate an outstanding program.

Reviewer: ```suggestion
What are you waiting for? Head over to [Sessionize](https://sessionize.com/iceberg-summit-2026/) to submit your proposal.
```

**Need help crafting your submission?** Join us for Speaker Office Hours in the [Iceberg Community Slack](https://join.slack.com/t/apache-iceberg/shared_invite/zt-287g3akar-K9Oe_En5j1UL7Y_Ikpai3A) #abstracts channel:

- Session 1: Wednesday, December 17, 2025 @ 8:00 AM PT
- Session 2: Wednesday, January 7, 2026 @ 8:00 AM PT
- Session 3: Wednesday, January 14, 2026 @ 8:00 AM PT

## What to Expect

The event will be primarily in-person with one track of content live-streamed. All non-workshop sessions will be recorded and made available after the event, ensuring the knowledge shared reaches the entire community.

## Spread the Word

Last year, we were blown away by the volume and quality of submissions from across the community. Let's make Iceberg Summit 2026 the best one yet! If you know someone doing exciting work with Iceberg, encourage them to submit a talk.

For more details about the event and to register your interest in attending, visit [icebergsummit.org](https://www.icebergsummit.org/).

> **See you in San Francisco!**

Reviewer: ```suggestion

Last year, we were blown away by the volume and quality of submissions from across the community. Let's make Iceberg Summit 2026 the best one yet! If you know someone working with Iceberg, encourage them to submit a talk and [register](https://www.icebergsummit.org/) to join us this April.

Can't make it in person? Be sure to register for a _free_ streaming pass to catch the keynotes and one track of content live. 

Hope to see you there!
```
